app/javascript/src/components/Board.vue

```diff
@@ -141,7 +141,9 @@ export default {
       if (this.paused) return false;
 
       if (this.game.availableActions) {
-        return [...this.game.availableActions].every((action) => action.type === 'buildFactory' || action.type === 'skipBuildFactory');
+        return [...this.game.availableActions].every(
+          (action) => action.type === 'buildFactory' || action.type === 'skipBuildFactory',
+        );
```

Comment:
Actually, can we remove this from the diff as well?

Reply:
I think it needed to be on an extra line cause lint was complaining, lint likes it the way it is now 

Tangentially related
[0cca894](https://github.com/Thrillberg/imperial/pull/711/commits/0cca8940a4eca8b10932615065951e453a9db159)